db/migration/000001_battleship.down.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE IF EXISTS analytics
```

Comment:
shouldn't this be `game_server_analytics`?

Reply:
woops! fixed

Reply:
the trick to avoid this, is to "redo" the migration. Meaning, run the `up.sql` script once then run the `down.sql` after. Most database clients, in our case `sqlc`, usually have a `redo` command.